images/krte/wrapper.sh

```diff
@@ -40,8 +40,9 @@ cleanup(){
   if [[ "${DOCKER_IN_DOCKER_ENABLED:-false}" == "true" ]]; then
     >&2 echo "wrapper.sh] [CLEANUP] Cleaning up after Docker in Docker ..."
     docker ps -aq | xargs -r docker rm -f || true
-    >&2 echo "wrapper.sh] [CLEANUP] Waiting for docker to stop for 30 seconds"
-    timeout 30 service docker stop || true
```

Comment:
so the issue is that it is not enough time to stop? Can it be reactivated somehow by something trying to open socket again? Will `start-stop-daemon` handle it?

I just don't know enough how `start-stop-daemon` work so and not many examples in k8s org.

Reply:
Yes I believe timeout is the issue , under the hood service stop ( used today ) calls start-stop-daemon . I confirmed this with strace. Proposal tries to kill the process after the timeout. 

With regard your second question , I don’t know will need to check and come back.

Reply:
Hi,

Performed a test manually in my local machine, `start-stop-daemon` will not handle it, it will just attempt to kill the process under /var/run/docker-ssd.pid. 

Checking the logs I see some errors in kubelet and i see that docker.log are stopped before the termination. I am not confident any longer that timeout is the issue, will abandon this PR and continue investigating the issue.

Reply:
/close